### ISSUE TITLE
Question about PointGroup

### ISSUE THREAD

**Issue by RayYoh** [Pointcept/Pointcept#99]
Dear authors,
Thanks for your excellent work _Pointcept_, and it is very helpful.
But a few details confused me when I read the code **PointGroup**. There is a ScoreNet in the original PointGroup code ([here](https://github.com/dvlab-research/PointGroup/blob/master/model/pointgroup/pointgroup.py#L170C19-L170C19)), and score loss after some epochs. But I didn't find the corresponding code in Poincept. Did I miss some critical code? Could you help me point out the relevant parts?

**Comment by Gofinge** [comment]
You didn't miss any part. In my reproduction, I found that the model can achieve better performance than the original paper even without such a mechanism (maybe due to better data processing). And I usually use it to validate the effectiveness of pre-trained model weight. Due to these two reasons, I didn't add the mechanism to our framework.
